reporter/html.js

```diff
@@ -599,7 +599,7 @@ export function escapeText( s ) {
 	}
 
 	function appendInterface() {
-		var qunit = id( "qunit" );
+		const qunit = id( "qunit" );
```

Comment:
nit: I believe this is not addressing any accessibility issue.

Reply:
True - you are correct ;-)
I try to cautiously fix as I go, just within the function block, if it doesn't sprawl on me.
If it would be more palatable, I could work through an independent change to convert as many `var` declarations as possible to `let`/`const`? That would keep changes more atomic with intent. What do you think?

Reply:
I did indeed say on a previous CR it was fine to change them all at once for a given area of code when already changing other variables inside of it.

I do agree though that it stands out here a bit unrelated as nothing else was changed regarding variables.

Doing them all at once, I wouldn't mind either. Maybe to keep the PR lighter to review, do it in two or three medium-size passes, e.g. `/src/cli`, `/src` (rest), `/test` and then misc files like `/runner` etc. And to keep down the need to rebase, I'd recommend doing one at a time, e.g. start the next after the previous one lands (unless you don't mind rebasing once or twice).

Thanks!

Reply:
Thanks for the suggestions - I'll prep some scoped changes for a few waves of churn.